.github/workflows/config.yml

```diff
@@ -0,0 +1,49 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
```

Comment:
Should rename this file to something like "pull_request_tests" or something? config is not so descriptive.